python-sdk/pyproject.toml

```diff
@@ -76,6 +76,10 @@ azure = [
     "azure-storage-blob",
     "smart-open[azure]>=5.2.1",
 ]
+sftp = [
+    "apache-airflow-providers-sftp>=4.0.0",
```

Comment:
Why is this restricted to `>=4.0.0` here, but not in `all` extra below?

Reply:
will add it there as well.